src/getter.js

```diff
@@ -102,6 +102,35 @@ function getStoreDeps(getter) {
   return storeDeps
 }
 
+/**
+ * Add override options to a getter
+ * @param {getter} getter
+ * @param {object} options
+ * @param {boolean} options.useCache
+ * @param {*} options.cacheKey
+ * @returns {getter}
+ */
+function addGetterOptions(getter, options) {
```

Comment:
@jordangarcia Is there an advantage to using a constructor outside of style preference?